### DISCUSSION TITLE
Outgoing links and unlinked mentions

### DISCUSSION THREAD

**Discussion by quesada** [zim-desktop-wiki/zim-desktop-wiki#1867] [Ideas]
This is one part of Obsidian that I would love to have on zim: 
https://help.obsidian.md/Plugins/Outgoing+links

Anyone thinking the same way?

I would never use obsidian because it's electron and typing latency is really high. But I really admire the community they formed (not being open source), the plugins they got going, and how fast they have grown.

**Comment by introt** [comment]
You can find outgoing links with the `LinksFrom` search keyword. Open `Search` > `Search backlinks...` (to avoid typing the name of the current page) and switch `LinksTo:` to `LinksFrom:`. These links can also be visualized using the LinkMap plugin. Distinguishing between links to existing and yet-to-be-created pages sounds useful for wikis with bigger indexes, and shouldn't be too hard to implement in a plugin similar to the one for BackLinks.

**Reply by introt** [reply]
Finding unlinked mentions by exact matching of page names isn't too hard either, but without page aliases it's usefulness is somewhat limited (one could also just use automatic `WikiLinks`). Would perhaps make a great pair with CAPTAGS and/or redirection pages (#1705).
